components/dropdowns/basic/src/index.scss

```diff
@@ -0,0 +1,119 @@
+/* sass-lint:disable class-name-format, no-important */
+@import '~@schibstedspain/theme-basic/lib/index';
+
+// TO-DO: Move it to theme basic.
```

Comment:
The idea is to extract some code here and move it to theme basic as well. That will be done once finished `DropdownUser` @JoanClaret is working on. Then, these 4 vars should be moved too.